changelog/1487.bugfix.1.rst

```diff
@@ -0,0 +1 @@
+Changed ``max_exp_bias - max_exp_bias`` to ``max_exp_bias - min_exp_bias`` inside `~plasmapy.diagnostics.langmuir.get_electron_temperature` when creating guess parameters for bimaxwellian fit function. Improves accuracy of piecewise linear fit.
```

Comment:
@rocco8773 Thanks for the suggestions. I committed the change for the reciprocal temperature bug. For your other suggestion, I'll also commit it, but as explanation, I tested the old and the revised fit function on a characteristic and plotted the fits:
![Graph of characteristic using buggy fit](https://user-images.githubusercontent.com/85201894/159840263-ba113879-fa1f-4dc0-9811-2d07c0d5f644.png)
![Graph of characteristic using fixed fit](https://user-images.githubusercontent.com/85201894/159840262-46b1f8e0-d7a5-495a-8a5b-9801d1ab0215.png)
It seems that sometimes, the bimaxwellian "knee" was placed outside of the bias range, so only one linear section was used. This is only anecdotal, though. It might be interesting to see how the calculated bimaxwellian temperatures change with this fix.